ToolBox/AlgoSeekOptionsConverter/Program.cs

```diff
@@ -29,72 +32,32 @@ public class Program
 
-            // first process tick/second/minute -- we'll do hour/daily at the end on a per symbol basis
-            var parallelism = 4;
-            var options = new ParallelOptions {MaxDegreeOfParallelism = parallelism};
-            var resolutions = new[] {Resolution.Tick, Resolution.Minute, Resolution.Second, Resolution.Hour, Resolution.Daily};
-            
-            // for testing only process the smallest 2 files
-            var files = Directory.EnumerateFiles(directory).OrderByDescending(x => new FileInfo(x).Length);
-            Parallel.ForEach(files, options, file =>
-            {
-                Log.Trace("Begin tick/second/minute/hour/daily: " + file);
-
-                var quotes = DataProcessor.Zip(dataDirectory, resolutions, TickType.Quote, true);
-                var trades = DataProcessor.Zip(dataDirectory, resolutions, TickType.Trade, true);
-
-                var streamProvider = StreamProvider.ForExtension(Path.GetExtension(file));
-                if (!RawFileProcessor.Run(file, new[] {file}, streamProvider, new AlgoSeekOptionsParser(), quotes, trades))
-                { 
-                    return;
-                }
-
-                Log.Trace("Completed tick/second/minute/hour/daily: " + file);
-            });
-            
-            Log.Trace("Begin compressing csv files");
-
-            var root = Path.Combine(dataDirectory, "option", "usa");
-            var fine =
-                from res in new[] {Resolution.Tick, Resolution.Second, Resolution.Minute}
-                let path = Path.Combine(root, res.ToLower())
-                from sym in Directory.EnumerateDirectories(path)
-                from dir in Directory.EnumerateDirectories(sym)
-                select new DirectoryInfo(dir).FullName;
+			var flushInterval = 1000000L;
 
-            var coarse =
-                from res in new[] {Resolution.Hour, Resolution.Daily}
-                let path = Path.Combine(root, res.ToLower())
-                from dir in Directory.EnumerateDirectories(path)
-                select new DirectoryInfo(dir).FullName;
+			Stopwatch sw = Stopwatch.StartNew();
+			AlgoSeekOptionsConverter.ConvertToFineResolution(sourceDirectory, destinationDirectory, flushInterval);
+			sw.Stop();
 
-            var all = fine.Union(coarse);
+//			git aAlgoSeekOptionsConverter.ExtractSymbol("goog", "/cache/TestData/RawAll/20151224", "/cache/TestData/RawGoog/20151224");
 
-            Parallel.ForEach(all, dir =>
-            {
-                try
-                {
-                    // zip the contents of the directory and then delete the directory
-                    Compression.ZipDirectory(dir, dir + ".zip", false);
-                    Directory.Delete(dir, true);
-                }
-                catch (Exception err)
-                {
-                    Log.Error(err, "Zipping " + dir);
-                }
-            });
+			Log.Trace(String.Format("Conversion finished in time: {0}", sw.Elapsed));
```

Comment:
Please restore the compression logic. This is needed to package up the csv files, I see a method linked in the other class but not called.